include/server.hpp

```diff
@@ -2,6 +2,8 @@
 
 #include <map>
 #include <string>
+#include <sys/types.h>
+#include <dirent.h>
```

Comment:
Is it allowed by the pdf?

Reply:
Yes, they are necessary for opendir and readdir which are allowed functions in the new version of the subject that got bonified about a month ago.